contrib/test/integration/README.md

```diff
@@ -1,21 +1,49 @@
 # Fedora and RHEL Integration and End-to-End Tests
 
 This directory contains playbooks to set up for and run the integration and
-end-to-end tests for CRI-O on RHEL and Fedora hosts. Two entrypoints exist:
+end-to-end tests for CRI-O on RHEL and Fedora hosts.  The expected entry-point
+is the ``main.yml`` Ansible playbook.
 
- - `main.yml`: sets up the machine and runs tests
- - `results.yml`: gathers test output to `/tmp/artifacts`
+The control-host:
```

Comment:
Unfortunately, the more bumbledeaded I become, the less I can imagine useful things like this 😦 

Excellent suggestions, happy to document them, thanks for pointing this out.  Exactly the type of feedback I needed.  Thanks.

Reply:
okay, I stuck two ``FIXME:`` into the file to remind me on these two points.  Until I get those written, to clarify the meanings:

**Control-host**: The system from which ``ansible-playbook`` is executed.
**Subject-host**: The target system(s) being manipulated by the playbook.

The way origin-ci works, it always assumes those two are exactly the same thing.  This results in some very strange / broken behavior if someone tries feeding the playbook an actual Ansible inventory-file.  e.g. The cri-o repo won't exist on any of the other systems because there never was a task to put it there (previously).